posthog/hogql/database/schema/test/test_cohort_people.py

```diff
@@ -0,0 +1,45 @@
+from posthog.hogql.parser import parse_select
+from posthog.hogql.query import execute_hogql_query
+from posthog.models import Person, Cohort
+from posthog.test.base import (
+    APIBaseTest,
+    ClickhouseTestMixin,
+)
+
+
+class TestCohortPeopleTable(ClickhouseTestMixin, APIBaseTest):
+    def test_select_star(self):
+        Person.objects.create(
+            team_id=self.team.pk,
+            distinct_ids=["1"],
+            properties={"$some_prop": "something", "$another_prop": "something1"},
+        )
+        Person.objects.create(
+            team_id=self.team.pk,
+            distinct_ids=["2"],
+            properties={"$some_prop": "something", "$another_prop": "something2"},
+        )
+        cohort1 = Cohort.objects.create(
+            team=self.team,
+            groups=[
+                {
+                    "properties": [
+                        {"key": "$some_prop", "value": "something", "type": "person"},
+                    ]
+                }
+            ],
+            name="cohort1",
+        )
+        cohort1.calculate_people_ch(pending_version=0)
```

Comment:
Very optional, to make it even more rock solid, you can 
```suggestion
        cohort1.calculate_people_ch(pending_version=0)
        cohort1.calculate_people_ch(pending_version=2)
        cohort1.calculate_people_ch(pending_version=4)
```

to simulate a few recalculations